src/components/App.tsx

```diff
@@ -2380,6 +2355,14 @@ class App extends React.Component<any, AppState> {
     let selectedElementWasDuplicated = false;
 
     const onPointerMove = withBatchedUpdates((event: PointerEvent) => {
+      if (dragOffsetXY === null) {
+        dragOffsetXY = getDragOffsetXY(
+          getSelectedElements(globalSceneState.getElements(), this.state),
+          originX,
+          originY,
+        );
+      }
```

Comment:
What we need is to get all selected elements, but if we try to `getSelectedElements` right in the `onPointerDown` handler, `this.state` won't reflect the changes to `state.selectedElementIds` we do in several places.

So I hacked it by getting the selected elements in the `onPointerMove` handler, hoping the `state` is updated by this point. Not sure if this is deterministic.